companyweb_base/views/companyweb_partner_view.xml

```diff
@@ -0,0 +1,139 @@
+                                attrs="{'invisible':[('cweb_creditLimit_enable', '!=', True )]}"
+                            />
+                            <field
+                                name="cweb_startDate"
+                                attrs="{'invisible':[('cweb_startDate_enable', '!=', True )]}"
+                            />
+                            <field
+                                name="cweb_endDate"
+                                attrs="{'invisible':[('cweb_endDate_enable', '!=', True )]}"
+                            />
+                            <field
+                                name="cweb_balance_year"
+                                attrs="{'invisible':[('cweb_balance_year_enable', '!=', True )]}"
+                            />
+                            <field
+                                name="cweb_equityCapital"
+                                attrs="{'invisible':[('cweb_balance_year_enable', '!=', True )]}"
+                            />
+                            <field
+                                name="cweb_addedValue"
+                                attrs="{'invisible':[('cweb_balance_year_enable', '!=', True )]}"
+                            />
+                            <field
+                                name="cweb_turnover"
+                                attrs="{'invisible':[('cweb_balance_year_enable', '!=', True )]}"
+                            />
+                            <field
+                                name="cweb_result"
+                                attrs="{'invisible':[('cweb_balance_year_enable', '!=', True )]}"
+                            />
+                            <field
+                                name="cweb_warnings"
+                                attrs="{'invisible':[('cweb_warnings_enable', '!=', True )]}"
+                            />
+                            <field
+                                name="cweb_url"
+                                attrs="{'invisible':[('cweb_url_enable', '!=', True )]}"
+                            />
+
+                            <field
+                                name="cweb_vat_liable"
+                                attrs="{'invisible':[('cweb_vat_liable_enable', '!=', True )]}"
+                            />
+                        </group> <!-- end of group financial -->
+                        <group name="cweb_group_barometer">
+                            <field
+                                name="cweb_image"
+                                attrs="{'invisible':[('cweb_score_enable', '!=', True )]}"
+                            />
+                        </group> <!-- end of group cweb_group_barometer -->
+                    </group> <!-- end of group bottom-->
+                </page>
+            </notebook>
+        </field>
+    </record>
+
+    <record id="companyweb_partner_view_button" model="ir.ui.view">
```

Comment:
Any particular reason to do a separate form for the button ?

Reply:
no, i've changed this